R/zchunk_L115.nh3_an_R_S_T_Y.R

```diff
@@ -115,13 +115,13 @@ module_emissions_L115.nh3_an_R_S_T_Y <- function(command, ...) {
 
     # Match total hybrid emissions data with EDGAR emissions data by technology
     # Create the from the scaler ratio of EDGAR total emissions to aggregated total hybrid emissions and multiplying by 1000
-    #
     L115.nh3_tg_R_G_sec_yr_tHyb %>%
       unite(col = "Region_GHG_Sector_Yr", c(GCAM_region_ID, Non.CO2, EDGAR_agg_sector, year), sep = "~") %>%
       left_join_error_no_match(unite(L115.EDGAR_R_G_sec_yr_v, col = "Region_GHG_Sector_Yr", c(GCAM_region_ID, Non.CO2, EDGAR_agg_sector, year), sep = "~"), by = "Region_GHG_Sector_Yr") %>%
       rename(EDGAR_emissions = value) %>%
       separate(Region_GHG_Sector_Yr, c("GCAM_region_ID", "Non.CO2", "EDGAR_agg_sector", "year"), sep = "~") %>%
-      mutate(., scaler = EDGAR_emissions / total_hybrid_emissions / 1000.0) ->
+      mutate(., scaler = EDGAR_emissions / total_hybrid_emissions / 1000.0,
```

Comment:
What on earth is this `.,` doing? It should be able to be removed. (I know this isn't _your_ code @russellhz )